src/reducers/projects.reducer.js

```diff
@@ -196,10 +244,8 @@ const prepareProjectForConsumption = (
     color: project.guppy.color,
     icon: project.guppy.icon,
     createdAt: project.guppy.createdAt,
-    // prettier-ignore
-    tasks: tasks
-      ? Object.keys(tasks).map(taskId => tasks[taskId])
-      : [],
+    // prettier-ignore $FlowFixMe
+    tasks: tasks ? Object.keys(tasks).map(taskId => tasks[taskId]) : [],
```

Comment:
Note: I have to check why `Object.keys` is required. Seems like the type for tasks is `Array<Task>` but I need to have a look if this is right. 

Or is my typing wrong here? For now I have added `$FlowFixMe`.

Reply:
OK, I've fixed the typing. `Array<Task>` is the return type from the function. As parameter it will receive `{start: {name: 'start', command: 'yarn start',..}, ... }`.